Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
this file should be in `.gitignore` (it's going to eventually contain your transifex session cookie)